File: app/src/main/java/com/projetdev/malo/musichall/models/Disc.java
package com.projetdev.malo.musichall.models;

public class Disc {

    private int id;
    private String name;
    private int dateReleased;
    private String thumbnail;
    private String label;

    public Disc(int id, String name, int dateReleased, String thumbnail, String label) {
        this.id = id;
        this.name = name;
        this.dateReleased = dateReleased;
        this.thumbnail = thumbnail;
        this.label = label;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public int getDateReleased() {
        return dateReleased;
    }

    public String getThumbnail() {
        return thumbnail;
    }

    public String getLabel() {
        return label;
    }
}

File: app/src/main/java/com/projetdev/malo/musichall/models/Disc.kt
package com.projetdev.malo.musichall.models

class Disc(
    override val id: Int, override val name: String, dateReleased: Int, override val thumbnail: String, artists: ArrayList<Artist>, style: List<String>?
) : Result(id, name, thumbnail, "", ""+dateReleased, style) {
    constructor(
        id: Int,
        name: String,
        dateReleased: Int,
        thumbnail: String,
        artists: ArrayList<Artist>,
        style: List<String>?,
        formats: List<String>?,
        country: String,
        url: String,
        labels: List<String>,
        trackList: ArrayList<Track>
        ) : this(id, name, dateReleased, thumbnail, artists, style)
}